utils/which/Makefile

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (C) 2010-2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
This should be removed or assigned to you.
https://github.com/openwrt/packages/blame/master/CONTRIBUTING.md#L30